Core/include/Acts/Surfaces/SurfaceArray.hpp

```diff
@@ -327,29 +267,24 @@ class SurfaceArray {
       return AnyGridConstView<SurfaceVector>{m_grid};
     }
 
-    const Transform3& getTransform() const override { return m_transform; }
-
-    Surface::SurfaceType surfaceType() const override { return m_type; }
-
-    /// @brief Get the number of dimensions of the grid.
-    /// @return number of dimensions
-    std::size_t dimensions() const override { return DIM; }
+    Surface::SurfaceType surfaceType() const override {
```

Comment:
I think I'm primarily using this for the detray conversion. Could you add a getter that allows accessing the surface instead? Then I can just access this (and will ultimately have to), and get the type from there. (Double check that I'm right and this is not otherwise used of course.)